WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+
+import { Button, Table } from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+import UserBadge from '../UserBadge';
+
+import '../../stylesheets/delegates/style.scss';
+
+const dasherize = (string) => string.replace(/_/g, '-');
+
+export default function Delegates({
+  delegates,
+  isEditVisible,
+}) {
+  const seniorDelegates = React.useMemo(() => delegates
+    .filter((user) => user.delegate_status === 'senior_delegate')
+    .sort((user1, user2) => user1.region.localeCompare(user2.region)), [delegates]);
+
+  // TO_VERIFY: I assume there are no cases where there are no delegates without
+  // senior delegate unless they are senior delegate
+
+  return seniorDelegates.map((seniorDelegate) => {
+    const delegatesUnderSenior = [seniorDelegate, ...delegates
+      .filter((user) => user.senior_delegate_id === seniorDelegate.id && user.delegate_status !== 'trainee_delegate')
+      .sort((user1, user2) => {
+        if (user1.region < user2.region) {
+          return -1;
+        } if (user1.region > user2.region) {
+          return 1;
+        } if (user1.name < user2.name) {
```

Comment:
nit: use `.localeCompare` for name comparisons.

Reply:
Done, thanks Cailyn.